WebClient/doc/core/window/windowManager/windowManager.md

```diff
@@ -58,3 +58,17 @@ Options object that should be passed to *WindowManager* constructor.
 | Event | Data | Description |
 |---|---|---|
 | dockingAvailabilityStatus | { dockingAvailabilityStatus: boolean } | Emitted when dock possibility changes. `data.dockingAvailabilityStatus` is `true` when docking is possible and `false` when not possible. |
+
+#### AddWindowOptions <a name="addWindowOptions"></a>
+
+| Property | Type | Description |
+|---|---|---|
+| position | [WindowPosition](#windowPosition) | Property indicates window position. |
+
+#### WindowPosition <a name="windowPosition"></a>
+
+*WindowPosition* is a string coinstant taht can be one of the following values.
```

Comment:
some minor spelling errors  in this line.

Reply:
👍